From 50c6f2693c08e41aac55909a8ba1edfc1e06076f Mon Sep 17 00:00:00 2001
From: Heinrich Toews <heinrich.toews@wago.com>
Date: Tue, 18 Jul 2017 11:59:46 +0000
Subject: [PATCH 081/292] dts: pfc200v3: cpsw: disable second rmii port to fix
 net traffic stagnation

After receiving an unspecific amount of packets the network interface
will stop to work on the receive channel.

The folling 'nping' command would freeze the interface:

sudo nping --icmp -N -H -c 50000000 --rate 10000 -e eth0 <ip_addr>

This is a workaround and does not remove the real problem.
The second interface has somehow influence on the behaviour of the
primary port.
---
 arch/arm/boot/dts/wago-pfc200v3.dtsi | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/arch/arm/boot/dts/wago-pfc200v3.dtsi b/arch/arm/boot/dts/wago-pfc200v3.dtsi
index a437ddb..ea5d2ff 100644
--- a/arch/arm/boot/dts/wago-pfc200v3.dtsi
+++ b/arch/arm/boot/dts/wago-pfc200v3.dtsi
@@ -849,14 +849,6 @@
  * Use a fixed phy for emac1.
  * that's rgmii2
  */
-&cpsw_emac0 {
-	phy-mode = "rmii";
-	fixed-link {
-		speed = <100>;
-		full-duplex;
-	};
-};
-
 &cpsw_emac1 {
 	phy-mode = "rmii";
 	fixed-link {
-- 
2.7.4

